### DISCUSSION TITLE
UIButton의 Style을 default로 바꾸면 image가 안뜨는데 왜 그럴까요??

### DISCUSSION THREAD

**Discussion by dudtjs1021ej** [swift-in-gangnam/soil#18] [Q&A] [answered]
<img src="https://user-images.githubusercontent.com/77915491/142203951-adce4fe7-24fd-4bb9-a5f8-319303e8f5f5.png" width="40%">
Button Style before iOS 15.0 [5] 이 경고 때문에 
Button Style을 default로 바꿨는데 이미지가 안떠요..🥲

<img src="https://user-images.githubusercontent.com/77915491/142204254-0f311c46-545b-433e-a631-062563b6fd3c.png" width="40%">

어떻게 해결하는지 아시는 분 있나요?

**Comment by Chuncheonian** [comment, marked as answer]
tintColor를 삭제하면 될 거 같아요.
그래도 안되면 background에 같은 이미지넣으면 될겁니다.

**Reply by dudtjs1021ej** [reply]
background 넣으니까 됐어요
이렇게 간단했다니.. 감사합니다 👍
